src/app/routes/api/v3/forms/__tests__/public-forms.verification.routes.spec.ts

```diff
@@ -432,6 +628,118 @@ describe('public-forms.verification.routes', () => {
       expect(response.body).toEqual(expectedResponse)
     })
 
+    it('should return 400 when using SingPass auth but user is not logged in to SingPass', async () => {
+      // Arrange
+      const expectedResponse = {
+        message: 'Sorry, something went wrong. Please refresh and try again.',
+      }
+
+      const mockJwt = 'mockJwt'
+
+      jest
+        .spyOn(SpOidcClient.prototype, 'verifyJwt')
+        .mockRejectedValueOnce(new Error())
+
+      // Act
+      const response = await request
+        .post(
+          `/forms/${mockSPFormId}/fieldverifications/${mockTransactionId}/fields/${mockEmailFieldId}/otp/generate`,
+        )
+        .send({
+          answer: 'mail@me.com',
+        })
+        // Note cookie is for SingPass, not CorpPass
+        .set('Cookie', [`jwtSp=${mockJwt}`])
+
+      // Assert
+      expect(response.status).toBe(StatusCodes.BAD_REQUEST)
+      expect(response.body).toEqual(expectedResponse)
+    })
+
+    it('should return 400 when using CorpPass auth but user is not logged in to CorpPass', async () => {
+      // Arrange
+      const expectedResponse = {
+        message: 'Sorry, something went wrong. Please refresh and try again.',
+      }
+
+      const mockJwt = 'mockJwt'
```

Comment:
supernit: seems to be repeated in many places, should we extract this into a constant?